README.md

```diff
@@ -1 +1,190 @@
-# Microverse-Portfolio-Project-
\ No newline at end of file
+# Microverse-Portfolio-Project-
+<div align="center"></div>
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] Kindly make sure to indent your **Table of content** properly so it looks much better as demonstrated in the readme template provided in the project requirement. Currently, as you can see it's not indented properly and it looks really messy and difficult to read.
<img width="979" alt="Screenshot 2023-05-30 at 19 05 09" src="https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/107192076/7d4cc03d-c16a-4744-90af-e56886045779">